src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -92,6 +94,14 @@ public Response getZipDownloadLimit() {
         return ok(zipDownloadLimit);
     }
 
+    @GET
+    @Path("exportFormats")
+    public Response getExportFormats() {
+        JsonArrayBuilder responseModel = Json.createArrayBuilder();
+        ExportService.getInstance().getExportersLabels().forEach(labels -> responseModel.add(Json.createObjectBuilder().add("displayName", labels[0]).add("formatName", labels[1])));
```

Comment:
Is displayName and formatName enough? While we're doing this, should we add isHarvestable, mediaType, and isAvailableToUsers (isVisibleInUserInterface might be better)?

For XML exporters there even these methods:

    public String getXMLNameSpace();

    public String getXMLSchemaLocation();

    public String getXMLSchemaVersion();